output/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
Can you add a quick comment at the top of this `.gitignore` explaining what it's accomplishing with its format?

Reply:
Removed this since we're committing the output.